geomesa-process/geomesa-process-vector/src/test/scala/org/locationtech/geomesa/process/transform/ArrowConversionProcessTest.scala

```diff
@@ -42,20 +44,30 @@ class ArrowConversionProcessTest extends Specification {
 
   "ArrowConversionProcess" should {
     "encode an empty feature collection" in {
+      ArrowAllocator.getAllocatedMemory mustEqual 0
```

Comment:
hmm, i don't think this is a safe check - even with `sequential`, the root allocator is shared across all unit tests, which often run in parallel. We could maybe modify the process method signature or constructor to pass in an optional allocator so that we can check it?

Reply:
Good eye!

Would we need to thread the optional allocator through to the SimpleFeatureVector / Writer classes?  (Or is that mostly already plumbed?)